docs/toolbar.md

```diff
@@ -0,0 +1,43 @@
+# Arduino Upload Package
+
+## Add a toolbar
+
+You can install `tool-bar` and `flex-tool-bar` Atom packages and use them to add a toolbar for Arduino commands, similar to the one in the official IDE.
+
+	apm install tool-bar
+	apm install flex-tool-bar
+	
+The following is an example for the Flex Tool Bar's configuration file, `toolbar.cson`. It adds the buttons for Arduino **Verify** and **Upload** commands (automatically saving the current file before), and another to open the **Serial Monitor**.
+
+	[
```

Comment:
This code would look nicer in a markdown code block with defined language 'coffee'